src/main/java/org/folio/rest/impl/BLUsersAPI.java

```diff
@@ -816,185 +825,198 @@ public void postBlUsersLogin(boolean expandPerms, List<String> include, String u
-        return;
-      }
-      //then get user by username, inject okapi headers from the login response into the user request
-      //see 'true' flag passed into the chainedRequest
-      userResponse[0] = loginResponse[0].thenCompose(client.chainedRequest(
-        userUrl, okapiHeaders, false, null, handlePreviousResponse(false,
-          false, true, aRequestHasFailed, asyncResultHandler)));
-
-      //populate composite based on includes
-      int includeCount = include.size();
-      ArrayList<CompletableFuture<Response>> requestedIncludes
-          = new ArrayList<>();
-      Map<String, CompletableFuture<Response>> completedLookup
-          = new HashMap<>();
-
-      for (int i = 0; i < includeCount; i++) {
-
-        if (include.get(i).equals(PERMISSIONS_INCLUDE)){
-          //call perms once the /users?query=username={username} (same as creds) completes
-          CompletableFuture<Response> permResponse = userResponse[0].thenCompose(
-                client.chainedRequest("/perms/users", okapiHeaders, new BuildCQL(null, "users[*].id", "userId"),
-                  handlePreviousResponse(false, true, true, aRequestHasFailed, asyncResultHandler)));
-          requestedIncludes.add(permResponse);
-          completedLookup.put(PERMISSIONS_INCLUDE, permResponse);
-        }
-        else if(include.get(i).equals(GROUPS_INCLUDE)){
-          CompletableFuture<Response> groupResponse = userResponse[0].thenCompose(
-            client.chainedRequest("/groups/{users[0].patronGroup}", okapiHeaders, null,
-              handlePreviousResponse(false, true, true, aRequestHasFailed, asyncResultHandler)));
-          requestedIncludes.add(groupResponse);
-          completedLookup.put(GROUPS_INCLUDE, groupResponse);
-        }
-        else if(include.get(i).equals(SERVICEPOINTS_INCLUDE)) {
-          CompletableFuture<Response> servicePointsResponse = userResponse[0].thenCompose(
-            client.chainedRequest("/service-points-users?query=userId=={users[0].id}" + QUERY_LIMIT,
-                okapiHeaders, null, handlePreviousResponse(false, false, false,
-                aRequestHasFailed, asyncResultHandler))
-          );
-          requestedIncludes.add(servicePointsResponse);
-          completedLookup.put(SERVICEPOINTS_INCLUDE, servicePointsResponse);
-          try {
-            CompletableFuture<Response> expandSPUResponse = expandServicePoints(
-              servicePointsResponse, client, aRequestHasFailed, okapiHeaders,
-              asyncResultHandler);
-            completedLookup.put(EXPANDED_SERVICEPOINTS_INCLUDE, expandSPUResponse);
-            requestedIncludes.add(expandSPUResponse);
-          } catch (Exception ex) {
-            client.closeClient();
-            asyncResultHandler.handle(io.vertx.core.Future.succeededFuture(
-              PostBlUsersLoginResponse.respond500WithTextPlain(ex.getLocalizedMessage())));
-            return;
-          }
-        }
-      }
```

Comment:
It would appear that none of the above lines have changed, yet they are included in this PR. Please only include what is actually necessary for this PR.

Reply:
This is just an example of unchanged lines being included. I suspect there are others....